db/migrate/20240304090449_migrate_interaction_settings_to_policy.rb

```diff
@@ -20,6 +20,9 @@ class NotificationPolicy < ApplicationRecord
   def up
     User.includes(account: :notification_policy).find_each do |user|
       deserialized_settings = Oj.load(user.attributes_before_type_cast['settings'])
+
+      next if deserialized_settings.nil?
```

Comment:
I realize this is committed already, but -- could we push this check into the initial User query, and only get records that are something like `where.not(settings: [nil, '', '{}'])` (or similar) ... or does that not work here b/c of the serialization on the column?